### PR TITLE
Revise upf.json configuration

### PR DESCRIPTION
**Pull request by osinstom**
This PR cleans up the structure of upf.json.

Modifications:
- create BESS-specific section and move all BESS-related parameters there
- use `fastpath` parameter instead of `enable_p4rt`, `enable_X`
- move all common parameters to the root of JSON file
- move all parameters specific to northbound control interface (PFCP/HTTP) to `cpiface`
- rename `p4rtciface` to `up4`

### CODE REVIEW

File: conf/upf.json [hunk 1]
```diff
@@ -133,19 +21,137 @@
         "n3_burst_bytes": 625000
     },
 
-    "": "Control plane controller settings",
+    "": "UPF fastpath to use (up4, bess, aether-bess)",
+    "fastpath": "bess",
```
Comment (pudelkoM) [inline conf/upf.json:25]: +1

**Comment by krsna1729** [comment]
#370

File: pfcpiface/bess.go [hunk 2]
```diff
@@ -644,12 +636,7 @@ func (b *bess) setUpfInfo(u *upf, conf *Conf) {
 	}
 
 	if conf.EnableEndMarker {
-		pfcpCommAddr := conf.EndMarkerSockAddr
```
Comment (osinstom) [inline pfcpiface/bess.go:647]: moved to `validate()`

File: pfcpiface/bess.go [hunk 3]
```diff
@@ -628,13 +625,8 @@ func (b *bess) setUpfInfo(u *upf, conf *Conf) {
 
 	b.client = pb.NewBESSControlClient(b.conn)
 
-	if conf.EnableNotifyBess {
-		notifySockAddr := conf.NotifySockAddr
```
Comment (osinstom) [inline pfcpiface/bess.go:632]: moved to `validate()`

File: conf/bess-upf.json [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+        "": "Max retries for sending a PFCP message from UPF",
+        "max_req_retries": 5,
+        "": "Timeout to wait for response from PFCP peer",
+        "resp_timeout": "2s",
+
+        "": "Period (in seconds) to wait until an inactive PFCP connection is closed",
+        "" : "read_timeout: 25",
+
+        "": "Whether to enable UPF HeartBeatTimer feature",
+        "enable_hbTimer": false,
```
Comment (ccascone) [inline conf/bess-upf.json:44]:
```suggestion
        "enable_hb_timer": false,
```

File: conf/bess-upf.json [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+        "": "Whether to enable Notify BESS feature",
+        "": "enable_notify_bess: false",
+        "": "Socket address for listening to Notify signals from the BESS pipeline",
+        "" : "notify_sockaddr: /tmp/notifycp",
+
+        "": "Socket address for sending End Markers",
+        "" : "endmarker_sockaddr: /tmp/pfcpport",
+
+        "": "Default QoS parameters for QFI values",
+        "qfi_qos_config": [
```
Comment (ccascone) [inline conf/bess-upf.json:141]: Isn't this common with up4? how do you configure burst sizes in up4?

**Comment by osinstom** [comment]
We decided to abandon this before SD-Fabric 1.1 release to avoid disruptive changes. We'll address that in the next release.

**Pull request closed without merging**